_attributes/common-attributes.adoc

```diff
@@ -59,6 +59,7 @@ ifdef::openshift-origin[]
 :rh-virtualization-engine-name: Engine
 endif::[]
 :launch: image:app-launcher.png[title="Application Launcher"]
+:mtc-first: {mtc-full} ({:mtc-short:})
```

Comment:
The first sentence of the below module is not replacing as expected. It is literally showing what you have here, not the replacements of it. I'm not really sure if you're able to use any of the attributes that you define in this same file.

Can you update this to just write out what you want?

```
mtc-first: Migration Toolkit for Containers (MTC)
```

![image](https://user-images.githubusercontent.com/7443887/175939772-7dff3a8d-a447-4f52-ac9a-6bdf0294fe14.png)